src/frontend/screens/Game/GamePage/components/MainButton.tsx

```diff
@@ -109,45 +163,60 @@ const MainButton = ({ gameInfo, handlePlay, handleInstall }: Props) => {
     )
   }
 
+  const handleAltLaunch = async () => {
+    setVerboseLogs(!verboseLogs)
+    await handlePlay(gameInfo)
+  }
+
   const is_installed = gameInfo.is_installed
 
   return (
-    <>
+    <div className="playButtons">
       {is_installed && !is.queued && (
-        <button
-          disabled={
-            is.reparing ||
-            is.moving ||
-            is.updating ||
-            is.uninstalling ||
-            is.syncing ||
-            is.launching ||
-            is.installingWinetricksPackages ||
-            is.installingRedist
-          }
-          autoFocus={true}
-          onClick={async () => handlePlay(gameInfo)}
-          className={classNames(
-            'button',
-            {
-              'is-secondary': !is_installed && !is.queued,
-              'is-success':
-                is.syncing ||
-                (!is.updating &&
-                  !is.playing &&
-                  is_installed &&
-                  !is.notAvailable),
-              'is-tertiary':
-                is.playing ||
-                (!is_installed && is.queued) ||
-                (is_installed && is.notAvailable),
-              'is-disabled': is.updating
-            },
-            'mainBtn'
+        <>
+          <button
+            className={classNames(
+              'button',
+              {
+                'is-secondary': !is_installed && !is.queued,
+                'is-success':
+                  is.syncing ||
+                  (!is.updating &&
+                    !is.playing &&
+                    is_installed &&
+                    !is.notAvailable),
+                'is-tertiary':
+                  is.playing ||
+                  (!is_installed && is.queued) ||
+                  (is_installed && is.notAvailable),
+                'is-disabled': is.updating
+              },
+              'mainBtn'
+            )}
+            disabled={
+              is.reparing ||
+              is.moving ||
+              is.updating ||
+              is.uninstalling ||
+              is.syncing ||
+              is.launching ||
+              is.installingWinetricksPackages ||
+              is.installingRedist
+            }
+            autoFocus={true}
+            onClick={async () => handlePlay(gameInfo)}
```

Comment:
These moved from before the `className` prop to after it. Not a problem, just wanted to mention it since it seems unintentional (nothing apart from the position was changed)